src/Views/Shared/SilenceBase.elm

```diff
@@ -0,0 +1,50 @@
+module Views.Shared.SilenceBase exposing (view)
+
+import Html exposing (Html, div, a, p, text, b)
+import Html.Attributes exposing (class, href)
+import Silences.Types exposing (Silence)
+import Types exposing (Msg(Noop, MsgForSilenceList))
+import Views.SilenceList.Types exposing (SilenceListMsg(DestroySilence))
+import Utils.Date
+import Utils.Views exposing (buttonLink)
+import Utils.Types exposing (Matcher)
+import Utils.List
+
+
+view : Silence -> Html Msg
+view silence =
+    let
+        f =
+            List.filter (\m -> m.name == "alertname") silence.matchers
+
+        alertName =
+            case List.head f of
+                Just m ->
+                    m.value
+
+                Nothing ->
+                    ""
```

Comment:
This would be a bit shorter:

```
alertName =
    silence.matchers
      |> List.filter (\{name} -> name == "alertname")
      |> List.head
      |> Maybe.withDefault ""
```